docs/docs/tutorials/get-started/customize-sign-in-experience.mdx

```diff
@@ -43,13 +44,79 @@ Toggle on "Enable terms of use" and enter the URL to enforce this process. Logto
 
 ### Language
 
-In most cases, it'll be fine to keep the default value unless:
+Logto provides six predefined languages (v1.0.0-beta.11). You can customize values of invidual keys.
```

Comment:
no need for mentioning the version. we'll have versioned docs later

```suggestion
Logto provides six predefined languages. You can customize values of individual keys.
```

Reply:
also i think this is unclear. what are values? and what are individual keys? please expand

Reply:
There is a terminology table to clarify all the definitions.

Reply:
> In most cases, it'll be fine to keep the default value unless:

Did you add this? @xiaoyijun

Reply:
I see, this is the old version... NM